src/Entities/AdditionalAddresses.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace GingerPluginSdk\Entities;
+
+use GingerPluginSdk\Collections\AbstractCollection;
+
+class AdditionalAddresses extends AbstractCollection
+{
+    public function __construct(Address $item)
```

Comment:
If you can have multiple addresses I think it would be nice if you can construct them correctly. The signature can be changed to:
public function __construct(Address... $addresses)

Reply:
Looks correct, will implement